### PR TITLE
Feature/o21 kb 23

### PR DESCRIPTION
**Pull request by manecipoglu**
This creates an endpoint on the inventory route that sends all inventory belonging to a specific warehouse.

### CODE REVIEW

**Comment by nilouhou** [comment]
@manecipoglu could you please change the base to Develop thanks

**Comment by manecipoglu** [comment]
> 

Yeah, I should have guessed something was wrong with the vast amount of commits. Should be good now.

File: server/routes/inventory.js [hunk 1]
```diff
@@ -5,48 +5,72 @@ const { readFile, writeFile } = require("../utils/helpers");
...
+    if (item.warehouseID === warehouseId) {
+      inventoryList.push(item);
+    }
+  });
+
+  res.status(200).json(inventoryList);
+});
 
 //Create a new inventory item
 inventoryRouter.post("/", PLACEHOLDER);
```
Comment (nilouhou) [inline server/routes/inventory.js:42]: Could you please get the latest, post is missing, thank you

**Comment by nilouhou** [comment]
Everything looks good to me, please resolve the conflict. thank you :)

File: server/routes/inventory.js (hunk 1 above)
Comment (manecipoglu) [reply server/routes/inventory.js:42]: I am afraid I can only do that from home. Will do at night. I also wouldn't mind if someone else pushes to my branch if you need to keep things rolling.

**Comment by nilouhou** [comment]
@manecipoglu could you please resolve the conflict, thanks

**Pull request merged**